### PR TITLE
HDDS-6453. Ozone start/stop script cannot resolve OM nodes in HA

### PR DESCRIPTION
**Pull request by HD372000**
## What changes were proposed in this pull request?
Change ozone getconf -ozonemanagers to output a list of all OMs as a plain list, one item per line (same as SCM).

## What is the link to the Apache JIRA
[HDDS-6453](https://issues.apache.org/jira/browse/HDDS-6453)

## How was this patch tested?
Tested in the cluster.

### CODE REVIEW

File: hadoop-ozone/common/src/test/java/org/apache/hadoop/ozone/TestOmUtils.java [hunk 1]
```diff
@@ -98,16 +98,16 @@ public void testGetOmHAAddressesById() {
     conf.set("ozone.om.address.ozone1.node2", "1.1.1.2");
     conf.set("ozone.om.address.ozone1.node3", "1.1.1.3");
     Map<String, List<InetSocketAddress>> addresses =
-        OmUtils.getOmHAAddressesById(conf);
+            OmUtils.getOmHAAddressesById(conf);
```
Comment (myskov) [inline hadoop-ozone/common/src/test/java/org/apache/hadoop/ozone/TestOmUtils.java:101]: I think it's better to remove indentation changes from this file.

**Review by myskov — COMMENTED** [review]
There's a unit test that tests class you made change to. See `testGetConfWithTheOptionOzoneManagers()`

**Review by adoroszlai — COMMENTED** [review]
Thanks @HD372000 for continuing work from #3366.

As @myskov mentioned, please avoid unnecessary whitespace changes.

Please check [CI results](https://github.com/HD372000/ozone/actions/runs/6087611488) for your change in your fork, there are a few failures.

[checkstyle](https://github.com/HD372000/ozone/actions/runs/6087611488/job/16516551725#step:6:525):

```
hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/conf/OzoneManagersCommandHandler.java
 48: Line is longer than 80 characters (found 82).
```

`TestGetConfOptions` is [failing](https://github.com/HD372000/ozone/actions/runs/6087611488/job/16516552656#step:6:2977) and needs to be updated.

[Failure](https://github.com/HD372000/ozone/actions/runs/6087611488/job/16517776926#step:5:367) of acceptance test for `getconf` command in non-HA environment indicates there is a regression in that case.

Failure in `TestOnDemandContainerDataScannerIntegration` seems unrelated, passed locally.

File: hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/conf/OzoneManagersCommandHandler.java [hunk 2]
```diff
@@ -41,10 +45,16 @@ public class OzoneManagersCommandHandler implements Callable<Void> {
   @Override
   public Void call() throws Exception {
     ConfigurationSource configSource =
-        OzoneConfiguration.of(tool.getConf());
+            OzoneConfiguration.of(tool.getConf());
```
Comment (hemantk-12) [inline hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/conf/OzoneManagersCommandHandler.java:48]: I think your IDE is not configured as per Ozone code style. Please use [ozone-style](https://github.com/apache/ozone/blob/764af7cf8dec5cc92e355b2d30339dee6b2c6493/hadoop-ozone/dev-support/intellij/ozone-style.xml#L14) if you are using IntelliJ.

**Review by adoroszlai — APPROVED** [review]

**Comment by adoroszlai** [comment]
Thanks @HD372000, @MLikeWater for the patch, @hemantk-12, @myskov for the review.

**Pull request merged**